trackalert/trackalert.conf

```diff
@@ -1,4 +1,10 @@
-webserver("0.0.0.0:8085", "--WEBPWD")
+-- Old way to configure the webserver
+-- webserver("0.0.0.0:8085", "--WEBPWD")
+-- New way to configure the webserver
+--          IP addr:port    Use SSL? Certificate File Private Key TLS Options - see https://www.openssl.org/docs/manmaster/man3/SSL_CONF_cmd.html
+addListener("0.0.0.0:8085", false,    "",             "",         {})
+setWebserverPassword("--WEBPWD")
```

Comment:
is that password what you wanted to use here?

Reply:
No, good catch, thanks!

Reply:
Oh, no this is trackalert.conf not trackers.conf.j2 - this is what I want here!